data/moves.ts

```diff
@@ -19436,13 +19436,27 @@ export const Moves: {[moveid: string]: MoveData} = {
 				}
 				return 5;
 			},
+			onModifyMove(move, source, target) {
+				if (move.useBaseOffensiveStatAndBoosts) {
+					if (move.useBaseOffensiveStatAndBoosts.includes("def")) {
+						move.useBaseOffensiveStatAndBoosts.replace("def", "spd");
+						if (source.boosts['def'] || source.boosts['spd']) {
+							this.hint(move.name + " uses Sp. Def boosts when Wonder Room is active.");
+						}
+					} else if (move.useBaseOffensiveStatAndBoosts.includes("spd")) {
+						move.useBaseOffensiveStatAndBoosts.replace("spd", "def");
+						if (source.boosts['def'] || source.boosts['spd']) {
+							this.hint(move.name + " uses Def boosts when Wonder Room is active.");
+						}
+					}
+				}
```

Comment:
id probably just slap an `if (!move.useBaseOffensiveStatAndBoosts) return;` at the top for less indentation

Reply:
Not needed. When using ?. it returns undefined which is treated as false. This makes it so I can remove that if statement altogether.

Reply:
it makes the code much more readable, even if it's not needed, which i think is preferable

Reply:
after seeing your suggested rework, I agree. I didn't get what you were suggesting at first.